test/test_evaluator.cpp

```diff
@@ -293,6 +293,28 @@ TEST_F(ExprEvalTest, Array) {
   checkIntValue(evaluator, bb, 5L);
 }
 
+TEST_F(ExprEvalTest, Struct) {
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+
+  auto* a = IrBuilder::create<Val>(DataType::Int);
+  auto* b = IrBuilder::create<Val>(DataType::Int);
+
+  auto struct_ = IrBuilder::structExpr({{"a", a}, {"b", b}}, "test_struct");
+
+  auto aa = IrBuilder::getAttrExpr(struct_, "a");
```

Comment:
nit: something like `IrBuilder::newStructExpr` seems more consistent with `IrBuilder::getAttrExpr`. No need to change.